_data/tools.yml

```diff
@@ -1538,6 +1538,15 @@
   link: https://42crunch.com/
   v2: true
   v3: true
+  
+- name: openapi-fuzzer
+  category: security
+  language: Rust
+  description:
+    Black-box fuzzer that fuzzes APIs based on OpenAPI specification.
```

Comment:
Can we try and explain what this fuzzer does without saying its a fuzzer that fuzzes? I know what fuzzing means in some contexts, but maybe not this one, and some people have never heard of fuzzing at all. Why do they need this in their life.

Reply:
@philsturgeon Thanks for the remark. How about this description?
"Based on OpenAPI specification, openapi-fuzzer provides random data as inputs to the API endpoins in order to find bugs"

Reply:
Brilliant yeah!

Reply:
Done :)